.swcrc

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://json.schemastore.org/swcrc",
```

Comment:
Seems to be a problem with the browserslist integration :/ 
https://github.com/swc-project/swc-loader/issues/37